workbook/components/session_service.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from django.utils import timezone
+
+from workbook.models.models import Session
+
+
+class SessionService:
+    def get_time_difference(self, session_start_time):
+        time_difference = timezone.now() - session_start_time
+        return time_difference.total_seconds() / 60
+
+    def is_valid(self, session_start_time):
+        return self.get_time_difference(session_start_time) < 1
```

Comment:
magic number, here is a perfect example, it is better to have it 
```suggestion
        return self.get_time_difference(session_start_time) < SESSION_TIMEOUT
```